notebooks/monte_carlo_case_study_nigeria_data.ipynb

```diff
@@ -0,0 +1,549 @@
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "from IPython.display import Image\n",
+    "import statsmodels.api as sm\n",
+    "from scipy.stats.mstats import zscore\n",
+    "from scipy import stats\n",
+    "from scripts.monte_carlo import (\n",
+    "            monte_carlo_sampling_pydoe2, monte_carlo_sampling_scipy, \n",
+    "            monte_carlo_sampling_chaospy, validate_parameters)\n",
+    "import logging\n",
+    "import yaml\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')\n",
+    "\n",
+    "pd.set_option('display.max_columns', 500)\n",
+    "\n",
+    "# disable logging from imported modules \n",
+    "logging.disable(sys.maxsize)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load the config file used for the monte carlo simulation\n",
+    "with open(\"mc/pypsa-earth/config.yaml\", \"r\") as stream:\n",
+    "    config = yaml.safe_load(stream)[\"monte_carlo\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Loading simulation results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/Users/gbotemi/Documents/code/PYPSA\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pwd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "All Networks loaded successfully\n"
+     ]
+    }
+   ],
+   "source": [
+    "# load solved networks into a list\n",
+    "network_list = []\n",
+    "\n",
+    "for count in range(0,49):\n",
+    "    # set directory of solved networks in results folder\n",
+    "    path = f\"mc/pypsa-earth/results/NG/networks/elec_s_10_ec_lcopt_Co2L-24H_m{count}.nc\"\n",
+    "    network = pypsa.Network(path)\n",
+    "    network_list.append(network)\n",
+    "\n",
+    "print(\"All Networks loaded successfully\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "objective loaded completely\n"
+     ]
+    }
+   ],
+   "source": [
+    "# load network objective values\n",
+    "network_obejective_list = []\n",
+    "\n",
+    "for idx in network_list:\n",
+    "    obj = idx.objective / 1e9\n",
+    "    network_obejective_list.append(obj)\n",
+    "print(\"objective loaded completely\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Reproducing the Parameters for Monte-Carlo Simulation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MONTE_CARLO_PYPSA_FEATURES = [\n",
```

Comment:
Have we inherited the capitalised variable names from some previous set-up of Monte-Carlo part? Not sure they look consistent...

Reply:
I could make it smaller cases. 

What I did was to copy the code implementation from the `monte_carlo.py` script